tests/integration/test_sns.py

```diff
@@ -65,6 +66,15 @@ def received():
             assert records[0][0]['Type'] == 'SubscriptionConfirmation'
```

Comment:
nitpick: Can we remove the `assert ...` statements with `self.AssertXXX(...)` as part of this PR? Would help clean up the code a bit (we should avoid using plain `assert` statements in the tests). Thanks